libr/core/cmd_flag.c

```diff
@@ -867,20 +867,30 @@ static int cmd_flag(void *data, const char *input) {
 				return 0;
 				}	
 			default:
-				addr = r_num_math (core->num, input + 2);
```

Comment:
uh?

Reply:
This line overrides the addr variable so that r_flag_get_at() would always return x0 as the flag. The addr variable is already set to core->offset at line 800. So I think the default case should just work like 'fd', which means: addr = core->offset.